src/constants/supportedNetworks.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ChainParams,
+  NetworkChains,
+  SupportedNetworks,
+} from "../types/supportedNetworks";
+
+export const SUPPORTED_NETWORKS: SupportedNetworks = {
+  Ethereum: {
+    icon: "images/ethereum-logo.png",
+    chainId: 1,
+  },
+  Avalanche: {
+    icon: "images/avalanche-logo.png",
+    chainId: 43114,
+  },
+  "BNB Chain": {
+    icon: "images/bnb-logo.png",
+    chainId: 56,
+  },
+  Polygon: {
+    icon: "images/matic-logo.png",
+    chainId: 137,
+  },
+};
+
+export const NETWORK_CHAINS: NetworkChains = {
+  "1": "Ethereum",
+  "43114": "Avalanche",
+  "56": "BNB Chain",
+  "137": "Polygon",
+  "5": "Goerli",
+};
+
+export const CHAIN_PARAMS: { [key: string]: ChainParams } = {
+  Avalanche: {
+    chainId: "0xa86a",
+    chainName: "Avalanche",
+    nativeCurrency: {
```

Comment:
ok never mind, it's beyond the scope of the ticket.

Reply:
Don't use `nativeCurrency`. I only asked if you could replace it.

Reply:
Now I'm only using `nativeCurrency` to get the logoUri. I consolidated all of the objects I made into `CHAIN_PARAMS`. I can't add `logoURI` into this object, otherwise Ethers will fail to add a new chain to a user's MetaMask if it doesn't have the network already